components/camel-slack/src/main/java/org/apache/camel/component/slack/SlackComponentVerifierExtension.java

```diff
@@ -118,23 +101,12 @@ private void verifyCredentials(ResultBuilder builder, Map<String, Object> parame
             String token = (String) parameters.get("token");
 
             try {
-                HttpClient client = HttpClientBuilder.create().useSystemProperties().build();
-                HttpPost httpPost = new HttpPost(parameters.get("serverUrl") + "/api/conversations.list");
-
-                List<BasicNameValuePair> params = new ArrayList<>();
-                params.add(new BasicNameValuePair("token", token));
-                httpPost.setEntity(new UrlEncodedFormEntity(params));
+                ConversationsListResponse response = Slack.getInstance(new CustomSlackHttpClient()).methods(token)
+                        .conversationsList(req -> req
+                                .types(Collections.singletonList(ConversationType.PUBLIC_CHANNEL))
```

Comment:
If we only have private channels, does still return `200`?

Reply:
Very good question, I created an empty slack workspace and for now it's impossible to remove all public channel. You can't delete the general one. 

So I tried with the opposite side, asked to list all private channels without having any and it replies 200 with this : 
```
{
    "ok": true,
    "channels": [],
    "response_metadata": {
        "next_cursor": ""
    }
}
```

Reply:
so I guess, it doesn't really makes difference since we just want to check if the token is valid